tests/integration/update-aos-ant-wasm.mjs

```diff
@@ -0,0 +1,35 @@
+import { AOS_MODULE_ID } from '@ar.io/sdk';
+import * as fs from 'node:fs';
+import path from 'node:path';
+
+const __dirname = path.dirname(new URL(import.meta.url).pathname);
+
+async function main() {
+  const fixturesDir = path.join(__dirname, 'fixtures');
+  if (!fs.existsSync(fixturesDir)) {
+    fs.mkdirSync(fixturesDir);
+  }
+  const fixtures = fs.readdirSync(fixturesDir);
+  const aosAntFiles = fixtures.filter(
+    (file) => file.startsWith('aos-ant-') && file.endsWith('.wasm'),
+  );
+
+  for (const file of aosAntFiles) {
+    if (!file.includes(AOS_MODULE_ID)) {
+      fs.unlinkSync(path.join(fixturesDir, file));
+    }
+  }
+
+  if (!aosAntFiles.some((file) => file.includes(AOS_MODULE_ID))) {
+    const res = await fetch(`https://arweave.net/${AOS_MODULE_ID}`).then((r) =>
```

Comment:
nit: inject gateway into the script via env var or similar and default to arweave.net

Reply:
https://github.com/ar-io/ar-io-sdk/pull/387/commits/01b1f96936d7654e6dcee3f5bb576b7c67e6846c